tests/ui/parallel-rustc/hello_world.rs

```diff
@@ -0,0 +1,6 @@
+// compile-flags: -Z threads=8
```

Comment:
What happens when these are run with the serial compiler?

Reply:
The `-Z threads` option has no influence with the serial compiler